ports/atmel-samd/boards/winterbloom_big_honking_button/pins.c

```diff
@@ -6,5 +6,7 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_GATE_OUT), MP_ROM_PTR(&pin_PA11) },
     { MP_ROM_QSTR(MP_QSTR_GATE_IN), MP_ROM_PTR(&pin_PA14) },
     { MP_ROM_QSTR(MP_QSTR_PITCH_IN), MP_ROM_PTR(&pin_PB08) },
+    /* Board revisions starting from v5 have PB10 tied to ground. */
+    { MP_ROM_QSTR(MP_QSTR_V5), MP_ROM_PTR(&pin_PB10) },
```

Comment:
Not sure why you want to name this pin. Is it so on earlier versions you can write code that sets its `.value` to `False`?

Reply:
Basically there will be a little check like this:

```python
v5_pin = digitalio.DigitalInOut(board.V5)
v5_pin.pull = digitalio.Pull.UP

if not v5_pin.value:
    # v5 or later
else:
    # v4 or earlier
```

v5 has a different front-end for the analog input that has a wider range (it's -5v to +5v instead of -2v to +2v), so I need to detect this in code so I can configure the ADC correctly.

Reply:
👍